cmd/rookd/api.go

```diff
@@ -30,21 +30,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var apiCmd = &cobra.Command{
-	Use:   "api",
-	Short: "Runs the Rook API service",
-}
-var apiPort int
-var repoPrefix string
+var (
+	apiCmd = &cobra.Command{
+		Use:   "api",
+		Short: "Runs the Rook API service",
+	}
+	apiPort    int
+	repoPrefix string
+	namespace  string
+)
 
 func init() {
 	apiCmd.Flags().IntVar(&apiPort, "port", 0, "port on which the api is listening")
 	apiCmd.Flags().StringVar(&repoPrefix, "repo-prefix", "quay.io/rook", "the repo from which to pull images")
-	apiCmd.Flags().StringVar(&cfg.containerVersion, "container-version", "latest", "version of the rook container to launch")
-	apiCmd.Flags().StringVar(&cfg.namespace, "namespace", "", "the namespace in which the api service is running")
+	apiCmd.Flags().StringVar(&cfg.versionTag, "version-tag", "latest", "version of the rook container to launch")
```

Comment:
Ahh cool.. Thanks for renaming this variable. VersionTag sounds better and more applicable :)